exporter/splunkhecexporter/client.go

```diff
@@ -78,6 +80,10 @@ func (b *bufferState) Read(p []byte) (n int, err error) {
 }
 
 func (b *bufferState) Close() error {
+	if b.closed {
+		return nil
+	}
+	b.closed = true
```

Comment:
Is this change really needed? Doesn't `gzip.Writer` already do that?

Reply:
I'm looking at https://github.com/golang/go/blob/master/src/compress/gzip/gzip.go#L228

Reply:
oh you got it indeed.